README.md

```diff
@@ -108,145 +40,59 @@ To proceed, click on the **Create a Bot** button and fill up all the required in
-
-Open the new file with your favourite text editor. If you still do not have one, take a quick look at [Atom](https://atom.io/), [Visual Studio Code](https://code.visualstudio.com/) or (in case you are a very brave developer) [Vim](https://www.vim.org). This is what the file contents should look like:
+To go to directory you have just cloned simply run `cd webexteams-python-example`. Try running `ls` and check that you can see all the files you will need to get your Bot up and running.
 
-````sh
----
-hello_bot:
-teams_access_token: <my_bot_access_token>
-````
-
-Yep, you guessed it! Replace `<my_bot_access_token>` with the Access Token you save during the **Create a Bot** step.
-
-### 5. Build Docker container
-
-Back to the terminal window: run the following command to build your Docker container.
+After installing Python, open terminal and run the following commands to install dependencies.
 
 ```sh
-./build.sh
+pip3 install -r requirements.txt
 ```
 
-After a couple of seconds, you should get a `Successfully built` message in your terminal window.
-
 <a name="run"></a>
 ## Run Bot
 
-### 1. Run Docker container
-
-If you are a **Windows user**, you will need to start with **Step 1**. If not, just skip this step and go directly to **Step 2**.
-
-**Step 1**
-
-Open **Notepad++** and open the file **run_hello_bot.sh**. You don't have Notepad++? Just quickly install it [here](https://notepad-plus-plus.org/).
-
-Once you have opened the file, go to the **Edit** menu on the top bar and click **EOL Conversion**. Set this to **Unix (LF)**. Then, save the file and close Notepad++.
-
-Repeat the above with **hello_bot/run.sh**.
+#### 1. Run Ngrok
 
-**Step 2**
-
-Run the Docker container with the following command.
+Copy the ngrok executable file that you downloaded above to the `webexteams-python-example` folder. On a terminal window, go to this directory and run the following command to expose a web server on port 5000 of your local machine to the internet.
 
 ```sh
-./run_hello_bot.sh
+./ngrok http 5000
 ```
 
-### 2. Interact with your Bot
-
-Login to your [Webex Teams](https://teams.webex.com/) account and **Create a space** by clicking the **+** button. Chose a name for the new space. Then, add your Bot to this space with the **Bot Username** you specified during the Create a Bot step. The format of this username should be **XXXX@webex.bot**.
-
-Say hello to your bot by typing and saying something `hello`. If the bot replies `hello, person who has email <your email>`, congratulations! You managed to set up your first Bot.
-
-<a name="understanding"></a>
-## Understanding the Bot
+#### 2. Configure Access Token
 
-### How does the bot know my personal email?
+Open `bot.py` with your favourite text editor. If you still do not have one, take a quick look at [Visual Studio Code](https://code.visualstudio.com/), [Atom](https://atom.io/) or (in case you are a very brave developer) [Vim](https://www.vim.org).
```

Comment:
hello_bot.py

Reply:
this was when I thought of changing the file name, but then agreed that it is better to keep it as it is :) thanks for noticing